conf/mesos-site.xml

```diff
@@ -74,7 +79,7 @@
 
   <property>
     <name>mesos.hdfs.executor.cpus</name>
-    <value>0.1</value>
+    <value>0.5</value>
```

Comment:
Why should the executor need this much cpu? It's not doing much beyond receiving fwkMessages and sending statusUpdates, right?